Xamarin.Forms.Platform.iOS/Cells/CellRenderer.cs

```diff
@@ -31,6 +40,15 @@ public virtual UITableViewCell GetCell(Cell item, UITableViewCell reusableCell,
 			return tvc;
 		}
 
+		void HandlePropertyChanged(object sender, PropertyChangedEventArgs e)
```

Comment:
I think that you can make it `virtual` and overload in successors as it is done in Android.

Reply:
That might be more futureproof indeed, done